src/p1/mod.rs

```diff
@@ -19,6 +19,9 @@ pub enum Error {
         line_number: usize,
     },
 
+    #[error("forbidden usage: {message}, doc: {doc_id}")]
+    ForbiddenUsage { message: String, doc_id: String },
```

Comment:
Include line number also here, else it would be hard for the author to figure out where the input it. Remember a ftd file may be 100s of lines long with multiple ftd.inputs.

Reply:
Ohh okay

Reply:
So the line number has to point the section (--ftd.input) or these headers (value/default-value) ?